cookbooks/aws-parallelcluster-install/recipes/cloudwatch_agent.rb

```diff
@@ -95,6 +99,6 @@
   dpkg_package package_path do
     source package_path
   end
-when 'amazon', 'centos'
+when 'amazon', 'centos', 'redhat'
```

Comment:
When there are if/else related to the Oses we are moving the code into a resource.

We should create a cloudwatch_agent resource, with two actions: setup and configure, move it in aws-parallelcluster-common/resources and call setup in the aws-parallelc-luster_install case and prepare in the config one.

Reply:
As discussed off-line, this will be part of a future task.